OgreMain/include/OgreTexture.h

```diff
@@ -125,6 +127,11 @@ namespace Ogre {
             const String& group, bool isManual = false, ManualResourceLoader* loader = 0);
 
         virtual ~Texture() {}
+
+
+        virtual uint getTextureId() {
```

Comment:
this is not needed: you can use `getCustomAttribute("GLID", ..)

Reply:
I do not understand, how can I get the GLID parameter in java code. Generated method takes `SWIGTYPE_p_void`, but I need to extract integer type

Reply:
yeah.. forgot about that, then you need your custom method

Reply:
added a wrappable version in #915